ReadMe.md

```diff
@@ -76,6 +76,28 @@ When you are ready to reset to the state just before installing:
 * stop the box with `vagrant suspend`, `vagrant halt`
 * delete the box with `vagrant destroy`
 
+### Upgrading the Testing Environment
+
+When bringing up your testing environment Vagrant may report that the box being used is out of date. You can also manually check to see if a newer box is available using the `vagrant box outdated` command.
+
+To upgrade the vagrant box used by your testing environment:
+
+ 1. Download the new box with `vagrant box update`
+     * **Note** as with the initial setup, this is a large download so please be patient
+ 1. Delete the exiting testing environment with `vagrant box destroy`

```

Comment:
```suggestion
 1. Delete the existing testing environment with `vagrant box destroy`
```

Reply:
```suggestion
 1. Delete the exiting testing environment with `vagrant destroy`
```

`vagrant box` does not have a destroy subcommand, the command wanted here to tear down the existing environment is just `vagrant destroy`

Reply:
Well both of those are embarrassing. I blame the 3am start 🤣